<launch>
    <!-- 加载YAML参数 -->
    <rosparam command="load" file="$(find ros_learning)/config/robot.yaml"/>

    <!-- 启动节点时可以引用这些参数 -->
    <node name="param_check_node" pkg="ros_learning" type="param_check.py" output="screen">
        <!-- 传递参数到节点 -->
        <param name="input_message" value="I am from launch file."/>
    </node>
</launch>